pinot-controller/src/main/java/org/apache/pinot/controller/helix/core/PinotHelixResourceManager.java

```diff
@@ -2573,6 +2577,13 @@ private void waitForSegmentsBecomeOnline(String tableNameWithType, Set<String> s
     return onlineSegments;
   }
 
+  public TableStats getTableStats(String tableNameWithType) {
+    String zkPath = ZKMetadataProvider.constructPropertyStorePathForResourceConfig(tableNameWithType);
+    ZNRecord znRecord = ZKMetadataProvider.getZnRecord(_propertyStore, zkPath);
+    String creationTime = SIMPLE_DATE_FORMAT.format(znRecord.getCreationTime());
```

Comment:
It occurred to me that it may be better to return a `long` , in seconds since epoch. Better granularity, less parsing and code on either side (read: easier for automation and scripts)

Reply:
I take that back. I think at this stage it is more useful to be able to readily read it on the console. We can leave it at this, but please do add seconds as well.

Reply:
Sure. Second is added now.